config/application.rb

```diff
@@ -25,8 +25,7 @@
 
 module RegisterTraineeTeachers
   class Application < Rails::Application
-    # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults(5.2)
+    config.load_defaults(7.0)
```

Comment:
If we use the `7.0` defafaults, then we should be able to remove the following files:

```
new_framework_defaults_5_2.rb
new_framework_defaults_6_0.rb
new_framework_defaults_7_0.rb
```

Any objections?